<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:5px;

</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>账号信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>个人信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:5px;
width:130px;
 height:70px;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>281</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="styleSheet">
       <string notr="true">border-radius:5px;

border:1 solid black;
height:20px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">border-radius:5px;

border:1 solid black;
height:20px;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">border-radius:5px;

border:1 solid black;
height:20px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>职业</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>学生</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>教师</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>学院</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;

border:1 solid #BDC0BA;
height:20px</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;

border:1 solid #BDC0BA;
height:20px</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(0, 185, 227);
border-radius:5px;
width:130px;
 height:70px;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>lineEdit_6</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
